### ISSUE TITLE
Translation to Danish (GUI)

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-gui#287]
For some messages in the GUI itself translation to Danish is incomplete. The following items in [src/assets/i18n/da.json] needs translation. Translation can either be provided as a pull request or as a comment to this issue.

In [src/assets/i18n/da.json]:
```json
    "Algorithm required": null,
    "All": null,
    "DELEGATED_PLURAL": null,
    "Digest type required": null,
    "NS required": null,
    "UNDELEGATED_PLURAL": null,
    "UNDELEGATED_SINGULAR": null,
```

When the key is a TAG then the translation should be a translation of the value in [src/assets/i18n/en.json]:
```json
    "DELEGATED_PLURAL": "Delegated",
    "UNDELEGATED_PLURAL": "Undelegated",
    "UNDELEGATED_SINGULAR": "Undelegated",
```

[src/assets/i18n/da.json]: https://github.com/zonemaster/zonemaster-gui/blob/develop/src/assets/i18n/da.json
[src/assets/i18n/en.json]: https://github.com/zonemaster/zonemaster-gui/blob/develop/src/assets/i18n/en.json

**Comment by matsduf** [comment]
Resolved by #293

**Comment by matsduf** [comment]
Resolved by #293
